lib/onboarding/gen_phrase/cubit/onboarding_gen_phrase_cubit.dart

```diff
@@ -41,7 +41,10 @@ class OnBoardingGenPhraseCubit extends Cubit<OnBoardingGenPhraseState> {
 
   final log = getLogger('OnBoardingGenPhraseCubit');
 
-  Future<void> generateSSIAndCryptoAccount(List<String> mnemonic) async {
+  Future<void> generateSSIAndCryptoAccount({
+    required List<String> mnemonic,
+    required bool restoreWallet,
```

Comment:
wallet restoration is not the purpose of generateSSIAndCryptoAccount. It should be handled in separate process

Reply:
`generateSSIAndCryptoAccount` is not called during the restoration process.